fastlane/Fastfile

```diff
@@ -166,7 +166,10 @@ lane :update_docs do
   template_path = File.expand_path("./assets/render_plugin.md.erb")
   clone_docs do
     docs_path = generate_markdown_docs(target_path: ".")
-    actions_md_path = File.join(docs_path, "actions.md")
+    actions_md_path = File.expand_path(File.join(docs_path, "docs/actions.md"))
```

Comment:
I'm not understanding why `actions.md` changes to `docs/actions.md`

Reply:
The return value was changed with a previous PR to not return the path to the directory containing `actions.md`, but the directory containing the `docs` folder, containing the `actions.md`. This is due to the fact that we now also copy resources (e.g. screenshots and logos), so we need to access those as well :)

Reply:
Ahhhh awesome